samples/subsys/mgmt/hawkbit/src/main.c

```diff
@@ -20,6 +21,40 @@
 
 LOG_MODULE_REGISTER(main);
 
+#ifdef CONFIG_HAWKBIT_CUSTOM_ATTRIBUTES
+struct hawkbit_cfg_data {
+	const char *VIN;
+	const char *customAttr;
+};
+
+struct hawkbit_cfg {
+	const char *mode;
+	struct hawkbit_cfg_data data;
+};
+
+static const struct json_obj_descr json_cfg_data_descr[] = {
+	JSON_OBJ_DESCR_PRIM(struct hawkbit_cfg_data, VIN, JSON_TOK_STRING),
+	JSON_OBJ_DESCR_PRIM(struct hawkbit_cfg_data, customAttr, JSON_TOK_STRING),
+};
+
+static const struct json_obj_descr json_cfg_descr[] = {
+	JSON_OBJ_DESCR_PRIM(struct hawkbit_cfg, mode, JSON_TOK_STRING),
+	JSON_OBJ_DESCR_OBJECT(struct hawkbit_cfg, data, json_cfg_data_descr),
+};
+
+int hawkbit_new_config_data_cb(const char *device_id, uint8_t *buffer, const size_t buffer_size)
+{
+	struct hawkbit_cfg cfg = {
+		.mode = "merge",
+		.data.VIN = device_id,
+		.data.customAttr = "Hello World!",
```

Comment:
good to attach a screenshot of this in the web UI

Reply:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/56227405/78eaf838-dc5e-4700-a62c-ccfc5fe5f3f8)